index.htm

```diff
@@ -459,21 +466,21 @@ <h5 v-for="coauthor in disc.coauthors" class="disc_coauthor">
 		    }
 		})
 		return fetched;
-	    },
+	    },*/
 	    flickrHandler: function (entry) {
 		entry.content = entry.content.replace('<p><a href=\"https://www.flickr.com/people/lapisniger/\">LapisNiger x y z t</a> ha postato una foto:</p>', '');
 		return entry;
 	    },
 	    instagramHandler: function (entry) {
 		entry.content = entry.content.replace(/<img src="https:\/\/scontent/g, '<img class="instagram-content" onclick="window.open(this.src)" src="https://scontent').replace(/\[组图\/Carousel\]/g, '').replace(/\[视频\/Video\]/g, '');
-		entry.title = entry.title.replace(/\[组图\/Carousel\]/g, '').replace(/\[视频\/Video\]/g, '');
+		entry.title = entry.title.replace(/\[组图\/Carousel\]/g, '').replace(/\[视频\/Video\]/g, '');/*
 		if (entry.title.endsWith('...')) {
```

Comment:
this is actually still needed,
consider this example
```
<item>
            <title><![CDATA[Ma se la carta è una Yama Gampi ricevuta da @pulppapier  e @patituccj , l'inchiostro è un Windsor e Newton ricevuto da @old_bezzy  e @nn.nnt  , i pe...]]></title>
            <description><![CDATA[<img src="https://scontent-iad3-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/s640x640/118403438_782595059232119_1634869602799828828_n.jpg?_nc_ht=scontent-iad3-1.cdninstagram.com&_nc_cat=111&_nc_ohc=IvrU44ybpK4AX9vL0vD&oh=6bcb2bbf58026fd8def0f11a708b88e8&oe=5F7CAC55" referrerpolicy="no-referrer"><p>Ma se la carta è una Yama Gampi ricevuta da @pulppapier  e @patituccj , l'inchiostro è un Windsor e Newton ricevuto da @old_bezzy  e @nn.nnt  , i pennelli sono un regalo di @dlupogram  e lo scorrimento si associa ai brani di Tim Hecker, io alla fine cosa ci ho messo? Un po' di rimestamento e la voglia di usare carte preziose improvvisando.

 #lapisniger   #brush   #yamagampi   #mulberrypaper   #ink   #impro   #giodi</p>]]></description>
            <pubDate>Sun, 30 Aug 2020 14:45:59 GMT</pubDate>
            <guid isPermaLink="false">https://www.picuki.com/media/2386545799571543028</guid>
            <link>https://www.picuki.com/media/2386545799571543028</link>
            <author><![CDATA[@lapisniger]]></author>
        </item>
```

The title ends with a truncated word `pe...` for `pennelli`

The content starts with the url of the image (this wasn't this way before) and then includes the full title

As we have little space in the widget, this snippet was removing the title from the content
you now need to remove the img tag too and to place it at the end of the content